src/dds/toolchain/from_json.cpp

```diff
@@ -76,6 +76,7 @@ toolchain dds::parse_toolchain_json_data(const json5::data& dat, std::string_vie
     opt_string     exe_prefix;
     opt_string     exe_suffix;
     opt_string_seq base_warning_flags;
+    opt_string_seq base_compile_flags;
```

Comment:
In order to support customization of `base_compile_flags` in a build which contains _both_ C and C++, there will need to be a `base_compile_flags` for each language, since it can vary by language (e.g. `/EHsc` for C++ on MSVC).

Reply:
Good catch. While one could use `c_flags`/`cxx_flags` to do this, I like this better and it's good to control those defaults separately (vs the all-or-nothing I initially had). Added `base_c_flags` and `base_cxx_flags`.